pkg/epinio/components/application/AppSource.vue

```diff
@@ -99,6 +99,7 @@ export default Vue.extend<Data, any, any, any>({
       },
 
       git: {
+        type:          sourceType,
```

Comment:
this gets persisted, so duplicates the type in the env var. what does it fix here?

Reply:
This is the type from data source and it's compared to the one from LabeledInput:
```
    if (this.value.type && this.value.type !== this.ghType) {
      return;
    }
```